posthog-android/src/main/java/com/posthog/android/internal/PostHogSharedPreferences.kt

```diff
@@ -108,10 +118,35 @@ internal class PostHogSharedPreferences(
         }
     }
 
+    private fun addToStringifiedKeys(key: String, editor: SharedPreferences.Editor) {
+        val stringifiedKeys = mutableSetOf<String>()
```

Comment:
in nit-picker mode I wonder if you could do this without allocating the mutable set since (pseudo code) really we just want `editor.putStringSet(STRINGIFIED_KEYS, getStringifiedKeys() + key)` 

but I don't remember Kotlin well enough to know if I'm just moving allocations around rather than reducing them so feel free to treat this as me thinking out loud and move on 😊

Reply:
The allocation is the same, but it is definitely less LoC, updated it to `getStringifiedKeys() + key`.